hack/tools/tools.go

```diff
@@ -1,5 +1,5 @@
-//go:build capibmtools
-// +build capibmtools
+//go:build tools || capibmtools
+// +build tools capibmtools
```

Comment:
I think we should just stick to either tools or capibmtools tag, we don't need both the tags. Lets fix the makefile where are using these tags.